10-Modelling-Deformable-Interfaces-with-Meshes/02-Vesicle-MD.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>Initializing a mesh object is new. This needs a little bit more description, explaining how the mesh properties (area, volume, etc..) are a function of both the vertices (particle positions) and the topology of the mesh. Also explain that the mesh topology is not stored in the gsd file and therefore needs to be provided separately.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>